docs/man-pages/condor_userprio.rst

```diff
@@ -114,7 +114,9 @@ Options
     *username* to the specified *value*.
  **-setceil** *username value*
     (Edit option) Set the ceiling for the user specified by
-    *username* to the specified *value*.
+    *username* to the specified *value*.  This value is
+    measured in *SlotWeight*, by default, the total number
+    of requested cores over all running jobs.
```

Comment:
SlotWeight should be a link to where ever we define slot weight.

I think it would also be clearer to rephrase this as something like: "This value is the sum of the SlotWeight of all running jobs.  By default, the slot weight of a running job is the number of cores allocated to that job." to make which part is the default definition of SlotWeight and which part is the definition of ceiling clear.